eng/pipelines/coreclr/jitstress-isas-sve.yml

```diff
@@ -15,6 +15,7 @@ pr:
     - src/coreclr/jit/emitarm64sve.cpp
     - src/coreclr/jit/emitfmtsarm64sve.h
     - src/coreclr/jit/lsraarm64.cpp
+    - src/coreclr/jit/lowerarmarch.cpp
```

Comment:
can you also add entries for following?
- lsraarmarch.cpp
- codegenarmarch.cpp

Reply:
Done.
Also alphabetically ordered the list.